changelog/1062.feature.rst

```diff
@@ -0,0 +1 @@
+Implement Guest invites.
```

Comment:
this could be more specific, e.g. link to `Invite.flags`

Reply:
d7f5150e4e991ba61851bcff4f67a74f8351b69b